app/tsconfig.json

```diff
@@ -24,6 +24,8 @@
     // "importHelpers": true,                 /* Import emit helpers from 'tslib'. */
     // "downlevelIteration": true,            /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
     // "isolatedModules": true,               /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
+    "skipLibCheck": true,
```

Comment:
Ant design icons seems to have issues with types, commented on PR https://github.com/ant-design/ant-design-icons/pull/299.

While this reduces the effectiveness of typescript, it also speeds up local compiling by skipping `node_modules` so we still get the benefit in our own codebase.